app/src/main/java/com/mk/tasky/core/domain/preferences/Preferences.kt

```diff
@@ -3,9 +3,9 @@ package com.mk.tasky.core.domain.preferences
 import com.mk.tasky.core.domain.model.LoggedUser
 
 interface Preferences {
-    fun saveToken(token: String)
-    fun saveFullName(name: String)
-    fun saveUserId(userId: String)
+    fun saveToken(token: String?)
+    fun saveFullName(name: String?)
+    fun saveUserId(userId: String?)
```

Comment:
What's the point of saving null values here? In case that relates to clearing them, I'd prefer having a function `deleteToken()` etc.
`saveToken(null)` doesn't really read as remove the token, at least not at the first sight

Reply:
Fixed!